CONTRIBUTING.md

```diff
@@ -106,7 +123,7 @@ Your contribution needs to meet the following standards:
   should be accompanied by a comment detailing its...
 
   - Justification, potentially including quantifiable reasons why safe
-    alternatives were not used (e.g. via a benchmark showing a valuable\[^1\]
+    alternatives were not used (e.g. via a benchmark showing a valuable[^1]
```

Comment:
this and below could be squashed in the previous commit.

Reply:
Done!